glean-core/android/src/test/java/mozilla/telemetry/glean/private/StringMetricTypeTest.kt

```diff
@@ -58,27 +56,6 @@ class StringMetricTypeTest {
         assertEquals("overriddenValue", stringMetric.testGetValue())
     }
 
-    @Ignore("Ignoring the test as the testing API for strings is not implemented")
-    @Test
-    fun `strings with no lifetime must not record data`() {
-        // Define a 'stringMetric' string metric, which will be stored in
-        // "store1". It's disabled so it should not record anything.
-        val stringMetric = StringMetricType(
-            disabled = true,
-            category = "telemetry",
-            lifetime = Lifetime.Ping,
-            name = "stringMetric",
-            sendInPings = listOf("store1")
-        )
-
-        // Attempt to store the string.
-        stringMetric.set("value")
-        // Check that nothing was recorded.
-        assertFalse("Strings must not be recorded if they have no lifetime",
-            stringMetric.testHasValue())
-    }
```

Comment:
Why is this test being deleted?  If it's not working yet (because `disabled` isn't implemented) maybe just update the `@Ignore` message?

Reply:
See commit message. There's no "no lifetime" thing and the test has the same contents as the following test (except its name)

Reply:
Ah -- missed that.  LGTM.